app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,78 @@
+.su-brand-bar__logo {
```

Comment:
Are you able to follow the procedure laid out here https://github.com/sul-dlss/component-library?tab=readme-ov-file#releasing ?  This is what we are doing in argo and purl.  If we have a common way of doing this, then it will be easier to manage in the long run.

Reply:
We don't have a base.html.erb template. I am guessing we will need one eventually so I can add it in but right now I am not sure how to do this without creating the file?